### DISCUSSION TITLE
Do releases happen on a schedule?

### DISCUSSION THREAD

**Discussion by rbiseck3** [pikepdf/pikepdf#574] [Q&A]
I'm using pikepdf and I need the version of `lxml` to be higher than version 5 when using `python3.9`, which was recently updated on `main`, but not yet in the latest release (`v8.14.0`). I haven't noticed a pattern on release timing. Or if possible, can a new release be requested to get that in?

**Comment by jbarlow83** [comment]
There's no schedule. 
The restriction on lxml was due to lxml not releasing certain binary wheels at the time 8.14 was released, so it prevented build and install failures.

The fix you reschedule has since between released.
